src/components/sidebarContent/bin/CommonDeletedItem.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div class="common-deleted-item">
+    <div class="title">
+      {{ item.title }}
+    </div>
+    <div class="deleted-at">
+      deleted at {{ item.deletedAt | getFormattedDate }}
+    </div>
+    <div class="buttons">
+      <ButtonText
+        text="restore"
+        small
+        :disabled="disabled"
+        @click="restore"
+      />
+      <ButtonText
+        text="delete"
+        style-type="underline"
+        small
+        :disabled="disabled"
+        @click="remove"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import ButtonText from '@/components/formElements/ButtonText.vue';
+
+export default {
+  props: {
+    type: {
+      type: String,
+      default: null,
+    },
+    item: {
+      type: Object,
+      default: null,
+    },
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  filters: {
+    getFormattedDate(val) {
+      const options = {
+        year: 'numeric',
+        month: 'numeric',
+        day: 'numeric',
+        hour: 'numeric',
+        minute: 'numeric',
+        second: 'numeric',
+        hour12: false,
+      };
+
+      return new Intl.DateTimeFormat('en', options).format(new Date(val));
+    },
+  },
+  components: {
+    ButtonText,
+  },
+  methods: {
+    restore() {
+      this.$emit('restore');
+    },
+    remove() {
```

Comment:
why not "delete"?

Reply:
because vue think that delete in template is a method for deleting object property